node/impl/full/eth.go

```diff
@@ -138,6 +139,10 @@ type EthModule struct {
 	EthTxHashManager         *EthTxHashManager
 	EthTraceFilterMaxResults uint64
 	EthEventHandler          *EthEventHandler
+
+	EthBlkCache   *arc.ARCCache[cid.Cid, *ethtypes.EthBlock]
```

Comment:
Can you please add some comments here.

so we can get clarity on why the same data is being stored on two different caches.